### PR TITLE
Made inline editing of task summary possible along with some keyboard…

### PR DESCRIPTION
**Pull request by marbetschar**
… shortcuts (ESC, Enter, Space).

This PR contains the first steps towards #54.

### CODE REVIEW

**Comment by marbetschar** [comment]
@danrabbit any suggested improvements to get this PR merged? Looking forward to tackle #69 based upon this work.

File: src/Widgets/TaskRow.vala [hunk 1]
```diff
@@ -130,8 +118,49 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
...
+                    if (has_focus) {
+                        show_popover ();
+                    } else {
+                        grab_focus ();
+                    }
+                    break;
+
+                case Gdk.Key.BackSpace:
+                    if (has_focus) {
+                        // TODO: Delete Task
```
Comment (danrabbit) [inline src/Widgets/TaskRow.vala]: In general, I would prefer if we didn't merge any dead code. We can use GitHub issues to describe ToDo's :)

Comment (marbetschar) [reply src/Widgets/TaskRow.vala]: If I got this straight you mean I should remove the `case Gdk.Key.BackSpace: ...` from the PR, right?

Comment (marbetschar) [reply src/Widgets/TaskRow.vala]: Dropped the case for BackSpace.

**Comment by marbetschar** [comment]
@danrabbit if you find some time it would be awesome to get this merged. Will tackle inline task editing next - based upon this work (#69).

**Pull request merged**